source/merge-deep.d.ts

```diff
@@ -0,0 +1,181 @@
+import {ConditionalExcept} from './conditional-except';
+import {FirstArrayElement, ArrayTail} from './internal';
+
+/**
+MergeDeep options.
+
+@param strict - Properties set to `undefined` value are skipped when set to `true` (default).
+
+@see MergeDeep
+*/
+export interface MergeDeepOptions {
+	strict?: boolean;
+}
+
+/**
+Represents an unknown record.
+*/
+type UnknownRecord = Record<string, unknown>;
```

Comment:
Shouldn't this be:

```suggestion
type UnknownRecord = Record<PropertyKey, unknown>;
```

?